docs/articles/documentation/using-testcafe/common-concepts/browser-support.md

```diff
@@ -5,57 +5,96 @@ permalink: /documentation/using-testcafe/common-concepts/browser-support.html
 ---
 # Browser Support
 
-TestCafe allows you to test against different web browsers.
-The most popular browsers are [supported directly](#directly-supported-browsers),
-while others require a [browser provider plugin](#browser-provider-plugins).
+* [Officially Supported Browsers](#officially-supported-browsers)
+* [Locally Installed Browsers](#locally-installed-browsers)
+* [Portable Browsers](#portable-browsers)
+* [Browsers on Remote Devices](#browsers-on-remote-devices)
+* [Browsers in Cloud Testing Services](#browsers-in-cloud-testing-services)
+* [Nonconventional Browsers](#nonconventional-browsers)
 
-This topic contains the following sections.
+## Officially Supported Browsers
 
-* [Directly Supported Browsers](#directly-supported-browsers)
-* [Browser Provider Plugins](#browser-provider-plugins)
-* [Specifying Browsers for Test Task](#specifying-browsers-for-test-task)
+While TestCafe is designed to support most of the modern browsers *de facto*, there is a limited number of *officially supported* browsers in which TestCafe is guaranteed to work.
 
-## Directly Supported Browsers
+* Chromium
```

Comment:
Are you sure that we should specify this?
We don't test hammerhead in Chromium browser.